dataplug-instagram/conf/silhouette.conf

```diff
@@ -5,7 +5,7 @@ silhouette {
   authenticator.cookiePath="/"
   authenticator.secureCookie=false // Disabled for testing on localhost without SSL, otherwise cookie couldn't be set
   authenticator.httpOnlyCookie=true
-  authenticator.useFingerprinting=true
```

Comment:
curious why was this change necessary?

Reply:
I was just testing to see if these settings make a difference for the bad signature exception. Answer is nope